src/main/java/com/google/devtools/build/lib/remote/BUILD

```diff
@@ -18,13 +18,22 @@ filegroup(
 
 java_library(
     name = "remote",
-    srcs = glob(["*.java"]),
+    srcs = glob(["*.java"], exclude = [
+        "ExecutionStatusException.java",
+        "ReferenceCountedChannel.java",
+        "RemoteRetrier.java",
+        "RemoteRetrierUtils.java",
+        "Retrier.java",
```

Comment:
That's surprising. Why do you need this?

Reply:
The `RemoteRetrier` class implements the logic of `--remote_retries`, which causes Bazel to retry failed gRPC calls sent to the remote executor/storage/downloader.

Reply:
Let me be more clear: This target looks odd. Why do you need to exclude the sources? Is there a way to have more meaningful targets rather than one large one?

Reply:
I would personally prefer smaller targets, and I also think it looks weird, but this is all pre-existing weirdness in Bazel.
* `//src/main/java/com/google/devtools/build/lib/remote` is a glob of every Java file in the directory.
* Since there is now a dependency of `RemoteModule` -> `downloader` -> `RemoteRetrier`, we need to have at least some of the Java files in `remote/` be a separate target.
* Other targets handle this case with globs that exclude some targets.

If you feel like adjusting the `remote/` directory to have finer-grained targets that sounds like a good idea to me. I would like to prefer doing any major surgery to the Bazel build graph as part of this PR.